drivers/clock_control/clock_control_nrf2_hfxo.c

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nordic_nrf_hfxo
+
+#include "clock_control_nrf2_common.h"
+LOG_MODULE_DECLARE(clock_control_nrf2, CONFIG_CLOCK_CONTROL_LOG_LEVEL);
+
+#include <hal/nrf_lrcconf.h>
+
+BUILD_ASSERT(DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) == 1,
+	     "multiple instances not supported");
+
+#define HFXO_START_TIME_US 900
```

Comment:
This value should be read from BICR. It's not constant. It depends on the installed xtal